src/components/timelapse/TimelapseBar.vue

```diff
@@ -45,7 +45,7 @@ export default {
   },
   methods: {
     advanceTimelapse () {
-      if (this.sliderVal == 12) {
+      if (this.sliderVal == this.maxVal) {
```

Comment:
Just noticed this, but we should be using `===` instead of `==` and `!==` instead of `!=` in this file. JS is weird.